client/src/components/comments/comment/index.tsx

```diff
@@ -0,0 +1,78 @@
+import { useTranslations } from 'next-intl';
+
+import { useUserInfo } from '@/context';
+import { Editor, UserIcon } from '@/components';
+
+import { IComment } from '../types';
+import { EditorBtnGroup } from '../btn-group';
+
+interface CommentListProps extends IComment {
+  onSave: () => void;
+  onDelete: (id: string) => void;
+  isEditing: string | null | boolean;
+  onDisabled: () => boolean | undefined;
```

Comment:
Instead of passing the onDisabled function, which gets executed during rendering, it's better to pass a precomputed boolean value isDisabled.